app/build.gradle

```diff
@@ -130,11 +130,15 @@ dependencies {
     implementation "com.squareup.retrofit2:adapter-rxjava2:$rootProject.retrofitVersion"
     implementation "com.squareup.okhttp3:logging-interceptor:$rootProject.okhttpVersion"
 
+    //scheduling jobs
+    implementation 'com.firebase:firebase-jobdispatcher:0.8.5'
+
```

Comment:
@teolemon I didn't notice but this requires google play services. Then we are left with android-Job and job-scheduler which does not require play services, but as we require compatibility till Sdk 16 we can only use android-job. So, do you want me to implement this?

Reply:
@tolemon what do you say?

Reply:
@teolemon  should I continue with this?

Reply:
I've done my research, firebase-jobdispatcher seems ok to use:
"Firebase JobDispatcher is an open-source library that provides an API similar to JobScheduler in the Android platform. Firebase JobDispatcher serves as a JobScheduler-compatibility layer for apps targeting versions of Android lower than 5.0 (API level 21).

Firebase JobDispatcher supports the use of Google Play services as an implementation for dispatching (running) jobs, but the library also allows you to define and use other implementations: For example, you might decide to use JobScheduler or write your own, custom code. Because of this versatility, we recommend that you use this Firebase JobDispatcher if your app targets a version of Android lower than 5.0 (API level 21)."

And https://gitlab.com/fdroid/fdroidserver/issues/429 mentions that it's ok, and that F-Droid has whitelisted the library.

Reply:
so, yes, you can solve the merge conflict, as suggested by @sachincool , and poke @policorn and @ross-holloway94